src/unix/notbsd/mod.rs

```diff
@@ -445,6 +445,16 @@ pub const IFF_PORTSEL: ::c_int = 0x2000;
 pub const IFF_AUTOMEDIA: ::c_int = 0x4000;
 pub const IFF_DYNAMIC: ::c_int = 0x8000;
 
+cfg_if! {
+    if #[cfg(target_os = "android")] {
+        pub const IFF_TUN: ::c_int = 0x0001;
```

Comment:
Could these be moved into the android folder to avoid the `cfg_if!`?

Reply:
okay, no problem.

by the way, should keep `IFF_TUN/IFF_TAP/IFF_NO_PI` type is same as linux ?

Reply:
Nah I think it's fine to fix the type for new definitions.

Reply:
ok, updated.